### PR TITLE
chore: Update GH-Template

### PR DESCRIPTION
**Pull request by salisdemos**
Update GH-Template

### CODE REVIEW

**Review by mussbach — DISMISSED** [review]

**Review by ahmad-demos — APPROVED** [review]
lgtm, codeReview!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,20 +1,30 @@
-<!-- **Ticket:** https://yaits.demos-deutschland.de/Txxyyzz -->
+### Ticket
+https://demoseurope.youtrack.cloud/issue/DPLAN-XXXXX
+or
+https://www.dev.diplanung.de/DefaultCollection/EfA%20DiPlanung/_workitems/edit/XXXXX
```
Comment (spiess-demos) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: I think that this is too much "internal" information, given that we're inside demosplan-core.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -29,3 +39,4 @@ Delete the checkbox if it doesn't apply/isn't necessary.
 - [ ] Update documentation
 - [ ] Link all relevant tickets
 - [ ] Move the tickets on the board accordingly
+- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))
```
Comment (spiess-demos) [inline .github/PULL_REQUEST_TEMPLATE.md:42]: I think we can use https://github.com/demos-europe/demosplan-documentation/blob/main/development/guidelines-conventions/coding-styleguides/twig_html.md#guideline-for-naming-cypress-hooks as a link, to give people outside our  vpn the possibility to read the docs.

**Review by spiess-demos — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (salisdemos) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: But we "always" put those Links, when we develop for projects managed in ADO. Is it more internal, than the YT-link?

Comment (gruenbergerdemos) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: YT is also automatically linked if we only write the ticket number (DPLAN-XXXX)

Comment (spiess-demos) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: agreed

Comment (salisdemos) [reply .github/PULL_REQUEST_TEMPLATE.md]: thanks. Makes it much easier 8077e147ca38c9b66217bd3443c30184d2b98437

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (salisdemos) [reply .github/PULL_REQUEST_TEMPLATE.md:42]: I would leave it like so. For our convenience.

**Pull request merged**